packaging/docker/fdb.bash

```diff
@@ -23,7 +23,7 @@
 source /var/fdb/scripts/create_server_environment.bash
 create_server_environment
 source /var/fdb/.fdbenv
-echo "Starting FDB server on $PUBLIC_IP:4500"
-fdbserver --listen_address 0.0.0.0:$FDB_PORT --public_address $PUBLIC_IP:4500 \
+echo "Starting FDB server on $PUBLIC_IP:${PUBLIC_PORT:-4500}"
```

Comment:
In the Dockerfile, the new entry is FDB_PUBLIC_PORT. We should update this script to match that naming.